internal/mocks/external/external.go

```diff
@@ -0,0 +1,3 @@
+// Package external implements mocks that implement 3rd party interfaces, t. i. interfaces which we do not control.
```

Comment:
I took the freedom and added a package comment in order to explain the `internal/.../external` situation.

Reply:
Thanks!